### DISCUSSION TITLE
"Forcing" blank lines

### DISCUSSION THREAD

**Discussion by PeterParslow** [agiorguk/gemini#94] [General]
This may only be an issue within tables (e.g. in page 1049).

Sometimes there are several paragraphs within a cell. With the cells being narrow, it isn't always clear when a new paragraph begins - so there are blank lines between the paragraphs.

This is simply achieved by having a blank link in the ASCIIdoc, which produces a blank line in the preview: but no in the HTML, where the blank line is suppressed.

I'll try
++++
<br/>
++++
but this gives two blank lines in the ASCII doc preview - let's see what it does in the HTML.

**Comment by archaeogeek** [comment]
Does this discussion help here? https://discuss.asciidoctor.org/Getting-blank-lines-in-AsciiDoc-td47.html

**Comment by PeterParslow** [comment]
My pair of ++++s has given the effect I wanted in the HTML
